FloconDesktop/composeApp/src/commonMain/kotlin/io/github/openflocon/flocondesktop/features/network/ui/NetworkUiState.kt

```diff
@@ -0,0 +1,25 @@
+package io.github.openflocon.flocondesktop.features.network.ui
+
+import androidx.compose.runtime.Immutable
+import io.github.openflocon.flocondesktop.features.network.ui.model.NetworkDetailViewState
+import io.github.openflocon.flocondesktop.features.network.ui.model.NetworkItemViewState
+import io.github.openflocon.flocondesktop.features.network.ui.view.filters.MethodFilter
+
+@Immutable
+data class NetworkUiState(
+    val items: List<NetworkItemViewState> = emptyList(),
```

Comment:
default params in view state is error prone, too easy to forget them during the implementation

easier to just not put them I think

Reply:
for preview, create a method `fun previewClassName()`